src/library/scala/collection/package.scala

```diff
@@ -133,14 +93,3 @@ package object collection extends LowPriority {
       else Some(t.init -> t.last)
   }
 }
-
-class LowPriority {
-  import scala.language.implicitConversions
-  import scala.collection._
-
-  /** Convert array to WrappedArray. Lower priority than ArrayOps */
-  def arrayToWrappedArray[T](xs: Array[T]): mutable.IndexedSeq[T] = mutable.WrappedArray.make(xs)
-
-  /** Convert String to Seq. Lower priority than StringOps */
-  def stringToSeq(s: String): immutable.WrappedString = new immutable.WrappedString(s)
-}
```

Comment:
I don’t think these conversions should go away. Are they duplicated by some other part of the code?

Reply:
This one is in [`Predef`](https://github.com/scala/scala/blob/v2.13.0-M4-pre-20d3c21/src/library/scala/Predef.scala#L612). The one in `Predef` has special null handling. Can't immediately say which would be better.